contracts/marketplace/src/multitest.rs

```diff
@@ -1430,7 +1431,7 @@ mod tests {
         let (marketplace, _) = setup_contracts(&mut router, &creator).unwrap();
 
         let update_config_msg = SudoMsg::UpdateParams {
-            trading_fee_percent: Some(5),
+            trading_fee: Some(5),
```

Comment:
this looks like an error. `Some(500)` now right?

Reply:
This is just a test to test updating so the value doesn't really matter.